### PR TITLE
Add RansomWhere.app v1.1.0

### PR DESCRIPTION
**Pull request by GregDurys**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
You need to read and fill out the template.

File: Casks/ransomwhere.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'ransomwhere' do
+  version '1.0.0'
+  sha256 '412a22a22342235f08bbe34512cc612ddeee11e79e1a89ba00e17b346c99a05c'
+
+  url "https://bitbucket.org/objective-see/deploy/downloads/RansomWhere_#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/ransomwhere.rb]: Needs a `# bitbucket.org/objective-see was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
This cask needs some uninstall cleanup.
image

**Comment by adidalal** [comment]
Closing due to lack of reply. Comment when changes are made and we'll be happy to reopen and take another look.

Thanks!

**Comment by ment4t** [comment]
Added the requested changes and bounced to 1.1.0.

**Comment by vitorgalvao** [comment]
This still does not clean the `launchd` daemons, [as pointed out](https://github.com/caskroom/homebrew-cask/pull/20722#issuecomment-214301748).

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in de646d41c55150f66025eada158ff0b0b0e84bae. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by ment4t** [comment]
Thanks. I probably have messed this up a bit when re-commiting, but I believe I added the cleanup  with commit fd1c3dfc245553c75cccb77e00095743fb2d3be2 ?

**Comment by jawshooah** [comment]
Note the changes made here: https://github.com/caskroom/homebrew-cask/blob/de646d41c55150f66025eada158ff0b0b0e84bae/Casks/ransomwhere.rb#L11-L21.

**Comment by ment4t** [comment]
Ya, these utilise the embedded install/uninstall script. I added the manual uninstall steps here: https://github.com/caskroom/homebrew-cask/commit/fd1c3dfc245553c75cccb77e00095743fb2d3be2. The end result is same. 
Will make it (more) hassle free with my next cask submission. Cheers.

**Comment by jawshooah** [comment]
Just FYI, the result is not the same since you didn't include an `uninstall :launchctl` directive, which kills the launch job. Deleting the daemon file is not enough.

**Comment by jawshooah** [comment]
Also, using `installer :script` instead of `app` skips the step of manually running the installer.

On a related note, if the installer did not provide a convenient CLI, `installer :manual` would be the appropriate stanza to use, not `app`.

**Comment by ment4t** [comment]
Gotcha. Will keep it in mind for next commits. Ta.

**Pull request closed without merging**